### ISSUE TITLE
Covers not downloading

### ISSUE THREAD

**Issue by oropeado** [djdembeck/Audnexus.bundle#74]
![Screenshot 2023-01-01 at 10-59-43 Plex](https://user-images.githubusercontent.com/75389565/210178887-bc368373-ea24-4439-bfb9-df20eb120f2f.png)

I had added new books to my Audiobook library on Plex and the new books do not download covers, but they'll download other metadata. Attached is what the poster page looks like. 
Things I have tried:
1. Refreshing Metadata
2. deleting the library and adding it again
3. clearing plugin cache
4. removing and reinstalling audnexus.bundle

I also use Prologue for the audiobooks and they don't pull the covers since there's none in the plex Metadata

**Comment by pawart-berger** [comment]
Hi @oropeado 
I have the same issue. How did you solve this?

I also can see some errors in the log everytime i try to refresh metadata:
[Req#71a] Error bringing media local (metadata://posters/com.plexapp.agents.audnexus_a8426870a85b72912b56eb3b1acee596ee9f25b6).

**Comment by theiamdude** [comment]
having similar issue, anyone has the steps to resolve this?

**Comment by pawart-berger** [comment]
> having similar issue, anyone has the steps to resolve this?

I found a solution for me:

I'm running Plex on Windows (not as a service) an changed the [Plex Data Directory](https://www.plexopedia.com/plex-media-server/windows/move-plex-data-directory-windows/) to another drive a few months ago.

After I tried some things and checked all the logs from Plex and also the Audnexus Plugin with multiple errors, I decided to change this Directory back to the original Path.

After this change, Covers download as they should.

I think this Path maybe is hardcoded in the Plugin.
